pallets/subtensor/src/weights.rs

```diff
@@ -448,28 +514,18 @@ impl<T: Config> Pallet<T> {
         uids.len() <= subnetwork_n as usize
     }
 
-    pub fn can_commit(netuid: u16, who: &T::AccountId) -> bool {
-        if let Some((_hash, commit_block)) = WeightCommits::<T>::get(netuid, who) {
-            let interval: u64 = Self::get_commit_reveal_weights_interval(netuid);
-            if interval == 0 {
-                return true; //prevent division by 0
-            }
-
-            let current_block: u64 = Self::get_current_block_as_u64();
-            let interval_start: u64 = current_block - (current_block % interval);
-            let last_commit_interval_start: u64 = commit_block - (commit_block % interval);
+    // TODO: check logic for this function
+    pub fn can_commit(netuid: u16, _who: &T::AccountId) -> bool {
+        let interval: u64 = Self::get_commit_reveal_weights_interval(netuid);
+        if interval == 0 {
+            return true; // prevent division by 0
```

Comment:
Is this an expected code path? Should it log a warning or something?

Reply:
done